boards/stm32f412gdiscovery/src/main.rs

```diff
@@ -7,14 +7,16 @@
 // https://github.com/rust-lang/rust/issues/62184.
 #![cfg_attr(not(doc), no_main)]
 #![deny(missing_docs)]
-
+#![feature(llvm_asm)]
```

Comment:
Why is this needed?

Reply:
I left it there by mistake, deleting it.